chain/actions/src/channels.rs

```diff
@@ -306,7 +306,7 @@ mod tests {
 
         let tx_sender = tx_queue.new_sender();
         async_std::task::spawn(async move {
-            tx_queue.action_loop().await;
+            tx_queue.start().await;
```

Comment:
This could call `start` directly, since it is a `spawn` within a `spawn` ?

Reply:
Actually, it should be the other way around... nice catch.